docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,273 @@ The parent key for all of the following parameters is `openshift4_local_storage`
 
 [horizontal]
 type:: string
-default:: `syn-openshift4-local-storage`
+default:: `openshift-local-storage`
 
 The namespace in which to deploy this component.
+Defaults to the suggested namespace for the RedHat Local Storage operator.
 
+== `local_storage_operator.channel`
 
-== Example
+[horizontal]
+type:: string
+default:: `"4.7"`
+
+The subscription channel to use when installing the Local Storage Operator using the Operator Lifecycle Manager.
+
+== local_volumes
+
+[horizontal]
+type:: dict
+keys:: names of `LocalVolume` resources to create
+values:: dicts with keys
+* `config` (mandatory)
+* `restricted_to` (optional)
+* `storage_class_devices` (mandatory)
+default::  `{}`
+
+The component will render a `LocalVolume` resource for each entry in the dict.
+
+=== Key `config`
+
+[horizontal]
+type:: dict
+keys:: `nodeSelector`, `tolerations`
+
+The value of key `config` of the entry is used as the base for field `.spec` of the `LocalVolume` resource.
+
+IMPORTANT: The component overwrites key `.spec.storageClassDevices` based on key <<_key_storage_class_devices,`storage_class_devices`>>.
+
+See the OpenShift documentation for the Local Storage Operator for possible configurations for
+
+* https://docs.openshift.com/container-platform/4.7/storage/persistent_storage/persistent-storage-local.html#local-volume-cr_persistent-storage-local[field `nodeSelector`]
```

Comment:
Maybe we can use `latest` instead of `4.7` to reduce the toll on maintaining this document.